jupyterhub/handlers/pages.py

```diff
@@ -591,6 +606,69 @@ def get(self, *args):
         self.finish()
 
 
+class Pagination(BaseHandler):
```

Comment:
Pagination isn't a handler:

```suggestion
class Pagination:
```

Reply:
let's also move this pagination class to its own top-level `jupyterhub/pagination.py` file. I think we'll likely want to use it on some API endpoints in the future like the user list.